github/actions_runners.go

```diff
@@ -314,3 +358,17 @@ func (s *ActionsService) RemoveOrganizationRunner(ctx context.Context, owner str
 
 	return s.client.Do(ctx, req, nil)
 }
+
+// RemoveEnterpriseRunner forces the removal of a self-hosted runner from an enterprise using the runner id.
+//
+// GitHub API docs: https://docs.github.com/en/rest/reference/enterprise-admin#delete-a-self-hosted-runner-from-an-enterprise
+func (s *ActionsService) RemoveEnterpriseRunner(ctx context.Context, enterprise string, runnerID int64) (*Response, error) {
+	u := fmt.Sprintf("enterprises/%v/actions/runners/%v", enterprise, runnerID)
+
+	req, err := s.client.NewRequest("DELETE", u, nil)
```

Comment:
This one appears to be new, but it should be moved alongside the other two endpoints listed above.

Reply:
done